src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -27,6 +28,7 @@
 import PIL
 import torch
 from huggingface_hub import model_info, snapshot_download
+from huggingface_hub.utils import send_telemetry
```

Comment:
Unfortunately `send_telemetry` is not yet released. It will be in `huggingface_hub>=0.13.0` which should be released soon. If you want to use it, you should wait for the release + pin a higher version of `hfh` (or add an alias to try to load `send_telemetry` but don't fail if not possible)